src/main/java/evaluation/metrics/GameMetrics.java

```diff
@@ -319,6 +319,53 @@ public Map<String, Class<?>> getColumns(int nPlayersPerGame, Set<String> playerN
         }
     }
 
+
+    public static class Actions2 extends AbstractMetric {
```

Comment:
Can we call this something more informative than Actions2 / add brief doc for why it's here?